sagify/template/sagify/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM python:{{ cookiecutter.python_version }}-slim-stretch
+ARG python_version
+FROM python:$python_version-slim-stretch
 
-MAINTAINER Kenza AI <support@kenza.ai>
+LABEL maintainer="Kenza AI <support@kenza.ai>"
```

Comment:
The `MAINTAINER` instruction has been deprecated; replaced it with Docker docs' suggested alternative.